e2e-playwright/src/app.e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { ProductMasterPage } from './page-objects/product-master.po';
+import { ProductDetailPage } from './page-objects/product-detail.po';
+
+describe('Product master page', () => {
+    let page: ProductMasterPage;
+    let detailPage: ProductDetailPage;
+
+    beforeEach(async () => {
+        page = new ProductMasterPage();
+        detailPage = new ProductDetailPage();
+    });
+
+    it('should display title', async () => {
+        await page.navigateTo();
+        expect(await page.getTitle()).toEqual('GET YOUR ICE CREAM');
+    });
+
+    it('should navigate to correct detail page when product was clicked', async () => {
+        await page.navigateTo();
+        const productName = 'Solero';
+        const productImage = await page.getProductImageByName(productName);
+
+        await productImage?.click();
```

Comment:
Minor:
`?.` sollte nicht nötig sein, da das Produkt Solero ja da sein sollte (sonst wäre der Test ja immer rot).
```ts
await productImage.click();
```

Reply:
Dadurch dass in `product-master.po.ts#getProductImageByName` nun nicht mehr eine Exception geworfen wird, kann dort im Promise u.U. `null` zurückkommen. Typescript bemerkt korrekt, dass `click` kein Property von `null` ist.

Reply:
OK.